src/plugins/simulator/components/simulator-tile.tsx

```diff
@@ -0,0 +1,61 @@
+import { VariableType } from "@concord-consortium/diagram-view";
+import { observer } from "mobx-react";
+import React, { useEffect, useState } from "react";
+
+import { SimulatorContentModelType } from "../model/simulator-content";
+import { ITileProps } from "../../../components/tiles/tile-component";
+import { BasicEditableTileTitle } from "../../../components/tiles/basic-editable-tile-title";
+
+import "./simulator-tile.scss";
+
+export const SimulatorToolComponent: React.FC<ITileProps> = observer((props) => {
+  // Note: capturing the content here and using it in handleChange() below may run the risk
+  // of encountering a stale closure issue depending on the order in which content changes,
+  // component renders, and calls to handleChange() occur. See the PR discussion at
+  // (https://github.com/concord-consortium/collaborative-learning/pull/1222/files#r824873678
+  // and following comments) for details. We should be on the lookout for such issues.
+  const { model, readOnly, scale } = props;
+  const content = model.content as SimulatorContentModelType;
+
+  const [_steps, setSteps] = useState(0);
+  useEffect(() => {
+    const id = setInterval(() => {
+      content?.step();
+      setSteps(v => v + 1);
+    }, content.simulationData.delay);
```

Comment:
If this runs on a readOnly component too then probably step is going to be called more often than you want when the left side of clue is open.

Reply:
I'm not sure of the best approach here. Turning this off in readOnly mode makes simulation tiles in curriculum not run, which I believe will be a problem because Leslie wants these simulation tiles to be omitted from the toolbar and only added to documents by dragging them from curriculum. Maybe simulation tiles in curriculum don't have to be running for this plan to work? But I think Leslie will expect them to be running.

Reply:
yes, I do. So yes, curriculum behavior is important.